internal/databases/greenplum/generic_meta_interactor_test.go

```diff
@@ -59,9 +59,23 @@ func TestFetch(t *testing.T) {
 	marshaller, _ := internal.NewDtoSerializer()
 	file, _ := marshaller.Marshal(testObject)
 	_ = folder.PutObject(internal.SentinelNameFromBackup(backupName), file)
-	backup, err := greenplum.NewGenericMetaFetcher().Fetch(backupName, folder)
+	actualResult, err := greenplum.NewGenericMetaFetcher().Fetch(backupName, folder)
+
+	//check equality of time separately
+	isEqualTimeStart := expectedResult.StartTime.Equal(actualResult.StartTime)
+	assert.True(t, isEqualTimeStart)
+
+	isEqualTimeFinish := expectedResult.FinishTime.Equal(actualResult.FinishTime)
+	assert.True(t, isEqualTimeFinish)
+
+	expectedResult.StartTime = time.Time{}
+	actualResult.StartTime = time.Time{}
+
+	expectedResult.FinishTime = time.Time{}
+	actualResult.FinishTime = time.Time{}
```

Comment:
```suggestion
	// since assert.Equal doesn't compare time properly, just assign the actual to the expected time  
	expectedResult.StartTime = actualResult.StartTime
	expectedResult.FinishTime = actualResult.FinishTime
```
```